docs/pages/attributesJSON.rst

```diff
@@ -230,3 +230,6 @@ The Light Setup attributes JSON should contain a single cell named "lights" that
 "type"
 	- string
 	- The type of the light.  "point" and "directional" are currently supported.
+"position_model"
+  - string
+  - They frame to use to place the light. "global", meaning stage's origin, and "camera", meaning place relative to a (potentially moving) camera, are currently supported.
```

Comment:
Oh, the "object" mode cannot be supported at the moment.

Reply:
yes, this would require more configuration specifications at the scene instance level, to specify which object instance the light should "orbit"

Reply:
Not "orbit". Just stay fixed relative to the object.
In that case, the "light" should be an object in the scene graph, and be attached to a child node of that object.

Reply:
Fixed orbit :).